python/ray/util/sgd/v2/backends/torch.py

```diff
@@ -140,3 +141,26 @@ def on_shutdown(self, worker_group: WorkerGroup,
         if len(worker_group) > 1:
             worker_group.execute(dist.destroy_process_group)
         worker_group.execute(shutdown_torch)
+
+    def convert_checkpoint(self, checkpoint: Dict) -> Any:
+        """Convert checkpoint to stream of bytes.
+
+        This prevents deserialization errors if driver is CPU only but
+        workers are on GPU.
+        """
+        _buffer = io.BytesIO()
+        torch.save(checkpoint, _buffer)
+        return _buffer.getvalue()
```

Comment:
Is there a more specific type hint that can be used for this return type?

Reply:
Should these be class methods? they don't rely on self